examples/__tests__/standard-ft/ft-tests.ava.js

```diff
@@ -0,0 +1,291 @@
+import { NEAR, Worker } from "near-workspaces";
+import test from "ava";
+
+const INITIAL_BALANCE = NEAR.parse("10000 N");
+const ONE_YOCTO = 1;
+const STOARAGE_BYTE_COST = 10_000_000_000_000_000_000n;
+const ACCOUNT_STORAGE_BALANCE = STOARAGE_BYTE_COST * 138n;
+
+test.beforeEach(async (t) => {
+    const worker = await Worker.init();
+    const root = worker.rootAccount;
+
+    const ftContract = await root.devDeploy("./build/my-ft.wasm");
+    await ftContract.call(
+        ftContract,
+        "init_with_default_meta",
+        {
+            owner_id: ftContract.accountId,
+            total_supply: INITIAL_BALANCE
+        }
+    );
+
+    const defiContract = await root.devDeploy("./res/defi.wasm");
```

Comment:
What is the defi contract? Could you briefly document its purpose and interface here?